jwst/tweakreg/tweakreg_step.py

```diff
@@ -278,7 +285,9 @@ def process(self, input):
         # can (and does) occur after alignment between groups
         if align_to_abs_refcat:
             try:
-                ref_image = images[0]
+                with images:
+                    ref_image = images.borrow(0)
+                    images.shelve(ref_image, 0, modify=False)
```

Comment:
I can't add a suggestion for the below unchanged lines. But what about moving the next (multi-line) statement into the `with images` block since `ref_image` is used?
At the moment either works. However if we decide to either hollow out models or explicitly close them on shelve we'll need to move the statement.